src/bm.h

```diff
@@ -1128,7 +1138,23 @@ void basm_translate_source(Basm *basm, String_View input_file_path, size_t level
                                 exit(1);
                             }
 
-                            basm_translate_source(basm, line, level + 1);
+                            int pos = sv_find_last_of(input_file_path, '/');
+                            if (pos < 0) {
+                                pos = sv_find_last_of(input_file_path, '\\');
+                            }
```

Comment:
It would be better to have some sort of abstraction called `Path` that
1. Abstracts away the platform specific path separators
2. Implements basic operations like `path_basename`, `path_filename`, etc.